libs/portals/admin/form-system/src/components/Navbar/components/NavComponent/components/NavButtons.tsx

```diff
@@ -0,0 +1,127 @@
+  const { groupsList: groups, inputsList: inputs } = form
+
+  const [addGroup] = useFormSystemCreateGroupMutation()
+  const [addInput] = useFormSystemCreateInputMutation()
+  const [removeStep, removeStepStatus] = useFormSystemDeleteStepMutation()
+  const [removeGroup, removeGroupStatus] = useFormSystemDeleteGroupMutation()
+  const [removeInput, removeInputStatus] = useFormSystemDeleteInputMutation()
+
+  const addItem = async () => {
+    if (activeItem.type === 'Step') {
+      const newGroup = await addGroup({
+        variables: {
+          input: {
+            groupCreationDto: {
+              stepId: activeItem?.data?.id,
+              displayOrder: groups?.length,
+            },
+          },
+        },
+      })
+      if (newGroup) {
+        controlDispatch({
+          type: 'ADD_GROUP',
+          payload: {
+            group: removeTypename(
+              newGroup.data?.formSystemCreateGroup,
+            ) as FormSystemGroup,
+          },
+        })
+      }
+    } else if (activeItem.type === 'Group') {
+      const newInput = await addInput({
+        variables: {
+          input: {
+            inputCreationDto: {
+              groupId: activeItem?.data?.id,
+              displayOrder: inputs?.length,
+            },
+          },
+        },
+      })
+      if (newInput) {
+        controlDispatch({
+          type: 'ADD_INPUT',
+          payload: {
+            input: removeTypename(
+              newInput.data?.formSystemCreateInput,
+            ) as FormSystemInput,
+          },
+        })
+      }
+    }
+  }
+
+  const remove = async () => {
+    const id = activeItem?.data?.id as number
+    if (activeItem.type === 'Step') {
+      await removeStep({
+        variables: {
+          input: {
+            stepId: id,
+          },
+        },
+      })
+      if (!removeStepStatus.loading) {
+        controlDispatch({ type: 'REMOVE_STEP', payload: { stepId: id } })
+      }
+    } else if (activeItem.type === 'Group') {
+      await removeGroup({
+        variables: {
+          input: {
+            groupId: id,
+          },
+        },
+      })
+      if (!removeGroupStatus.loading) {
+        controlDispatch({ type: 'REMOVE_GROUP', payload: { groupId: id } })
+      }
+    } else if (activeItem.type === 'Input') {
+      await removeInput({
+        variables: {
+          input: {
+            inputId: id,
+          },
+        },
+      })
+      if (!removeInputStatus.loading) {
+        controlDispatch({ type: 'REMOVE_INPUT', payload: { inputId: id } })
+      }
+    }
+  }
+
+  return (
+    <Box display="flex" flexDirection="row">
+      {activeItem.type !== 'Input' && (
+        <Box
+          style={{ paddingTop: '5px', cursor: 'pointer' }}
+          marginRight={1}
+          onClick={addItem}
+        >
+          <Icon icon="add" color="blue400" size="medium" />
+        </Box>
+      )}
+      <Box style={{ paddingTop: '5px', cursor: 'pointer' }} onClick={remove}>
```

Comment:
Again this prop are available on the box component. Please review the rest of the PR with that in mind.

Reply:
Please review this. Also the design system uses a spaces which are multiples of 8px. So `paddingTop = {1}` is 8px, couldn't you just use that instead?